rocAL/rocAL_pybind/examples/tf_petsTrainingExample/requirements.sh

```diff
@@ -0,0 +1,8 @@
+# installing required packages
+pip install contextlib2
```

Comment:
@kiritigowda Can you add this line in the PR itself as line 1 in the file. Should fix the issue
```
#!/bin/bash
```